two-scale-heat-conduction/macro-dumux/appl/main.cc

```diff
@@ -297,22 +305,17 @@ int main(int argc, char **argv)
     // advance precice
     if (getParam<bool>("Precice.RunWithCoupling") == true) {
       couplingParticipant.advance(dt);
```

Comment:
`nonLinearSolver.solve(x, *timeLoopo);` may solve for a smaller time-step size than requested if the first attempt fails. The following fix does not account for the coupling data being different for this new time, follow-up required to read appropriate data in newton solver.
```suggestion
      couplingParticipant.advance(timeLoop->timeStepSize());
```